config_repo/settings_ZWO.json.repo

```diff
@@ -60,7 +60,7 @@
    "debuglevel":"1",
    "newexposure":"0",
    "alwaysshowadvanced":"0",
-   "showonmap":0,
+   "showonmap":1,
```

Comment:
@EricClaeys  change newexposure  != showonmap
mistakenly changed?

Reply:
How in the world did I  mess that up?  Thanks for catching it.